test/e2e/test_app.py

```diff
@@ -109,13 +109,13 @@ def test_complete_app(browser, workflow_with_customizations, testing_env_variabl
 
     # Find the base test asset in the collection
     # FIXME - it would be better to find the workflow with the correct assetId, but I can't figure out how to do it with selenium.  
-    # Instead, we ae taking the first workflow in the list and assume it is the one for the test
+    # Instead, we are taking the second workflow in the list and assume it is the one for the test
```

Comment:
Curious, why are we taking the second workflow now?

I think I understand why and I think it makes sense but I just want to confirm my assumption.

Is it because the workflow creates two identical workflows but only waits on the first one so we really want to be running our tests on the first one but the web page lists the workflows in order from newest on the top to oldest on the bottom? So, I think the second workflow shown on the page will be the first of two that we created. Is that my understanding correct?

Reply:
I meant "...test case creates two identical workflows..."

Reply:
There's also another workflow that gets created after this one and has different settings. Now this workflow is the second on the list.
I'm not sure when this change occurred, but it's no longer the first on the list (last workflow to be created)

Reply:
There should be three workflows after both test cases run.
This test case creates 2 workflows that are exactly the same, as far as I can tell.
The other test case creates 1 additional workflow with different settings.

But, each test case runs synchronously. i.e., the first test case creates its 2 workflows then runs its validations, then the second test case creates 1 additional workflow then runs its validations. So, when the first test case runs, there are only 2 workflows (both the same). The third workflow is created by the second test case after the first test case has completed and that becomes the top workflow in the list.

So, at the time this code is running, I believe the last workflow to be created hasn't yet been created and the first two workflows in the list will be the two identical workflows created by this test case.

That all said, if you set environment variable `USE_EXISTING_WORKFLOW=True` so it doesn't create new workflows and you previously ran both test cases, the first workflow will be the one created for the second test case. So, I believe your change will make it work better. In any case, I think the change is good.